app/content/drill/[id]/leaderboard.js

```diff
@@ -177,10 +178,18 @@ export default function Leaderboard() {
             >
               <List.Item
                 title={userInfo[userId] ? userInfo[userId]["name"] : "Unknown"}
-                left={() => <Avatar.Text size={24} label="XD" />}
+                left={() => (
```

Comment:
In my PR #161, I added numbers to the leaderboard which used this section too so we just need to be mindful when merging both of them to keep the functionality of both changes.